quickcheck-dynamic-iosim/quickcheck-dynamic-iosim.cabal

```diff
@@ -41,6 +41,20 @@ common lang
     ImportQualifiedPost
     ScopedTypeVariables
     StandaloneDeriving
+    DerivingStrategies
+    FlexibleContexts
+    FlexibleInstances
+    GADTs
+    KindSignatures
+    MultiParamTypeClasses
+    TupleSections
+    TypeFamilies
+    TemplateHaskell
+    TypeApplications
+    ConstraintKinds
+    DataKinds
+    PolyKinds
+    RankNTypes
 
   ghc-options:
     -Wall -Wnoncanonical-monad-instances -Wunused-packages
```

Comment:
While we are at it, how about `-Werror`?

Reply:
I vote `-Werror` in CI but not during development?